AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -1,122 +1,165 @@
-c. Enter `ssh -N -f -L localhost:8888:localhost:8888 <userid@hostname>`<br>
-d. Copy-paste the URL address from the host into your local browser to open the jupyter console.<br>
-3. Go to `ResNet50_Inference.ipynb` and run each cell to create synthetic data and run int8 inference.
-
----
-**NOTE**
+#### Activate Conda without Root Access (Optional)
 
-In the jupyter page, be sure to select the correct kernel. In this example, select 'Kernel' -> 'Change kernel' -> Python [conda env:tensorflow].
+You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
 
----
+```
+conda create --name user_tensorflow --clone tensorflow
+conda activate user_tensorflow
+```
 
-### **Request a Compute Node**
-In order to run on the DevCloud, you need to request a compute node using node properties such as: `gpu`, `xeon`, `fpga_compile`, `fpga_runtime` and others. For more information about the node properties, execute the `pbsnodes` command.
- This node information must be provided when submitting a job to run your sample in batch mode using the qsub command. When you see the qsub command in the Run section of the [Hello World instructions](https://devcloud.intel.com/oneapi/get_started/aiAnalyticsToolkitSamples/), change the command to fit the node you are using. Nodes which are in bold indicate they are compatible with this sample:
+#### Navigate to Model Zoo
 
-<!---Mark each compatible Node in BOLD-->
-| Node              | Command                                                 |
-| ----------------- | ------------------------------------------------------- |
-| GPU               | qsub -l nodes=1:gpu:ppn=2 -d . hello-world.sh           |
-| CPU               | qsub -l nodes=1:xeon:ppn=2 -d . hello-world.sh          |
-| FPGA Compile Time | qsub -l nodes=1:fpga\_compile:ppn=2 -d . hello-world.sh |
-| FPGA Runtime      | qsub -l nodes=1:fpga\_runtime:ppn=2 -d . hello-world.sh |
+Navigate to the Model Zoo for Intel® Architecture source directory. By default, it is in your installation path, like `/opt/intel/oneapi/modelzoo`. 
 
+1. View the available Model Zoo release versions for the AI Kit:
+   ```
+   ls /opt/intel/oneapi/modelzoo
+   1.8.0  latest
+   ```
+2. Navigate to the [Model Zoo Scripts](https://github.com/IntelAI/models/tree/master/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
+   ```
+   cd /opt/intel/oneapi/modelzoo/latest
+   ```
 
-### Troubleshooting
-If an error occurs, troubleshoot the problem using the Diagnostics Utility for Intel® oneAPI Toolkits.
-[Learn more](https://software.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html)
+#### Install Jupyter Notebook
 
-### Using Visual Studio Code*  (Optional)
+```
+conda install jupyter nb_conda_kernels
```

Comment:
This step *fails* in DevCloud because my instance runs out of memory. It looks like I need to run a job on qsub? But that doesn't make sense for an installation.....
```(tensorflow) u164637@login-2:/opt/intel/oneapi/modelzoo/latest$ conda install jupyter nb_conda_kernels
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): - ERROR conda.auxlib.logz:stringify(163):
Traceback (most recent call last):
  File "/glob/development-tools/versions/oneapi/2023.0.1/oneapi/intelpython/latest/lib/python3.9/site-packages/conda/auxlib/logz.py", line 157, in stringify
    requests_models_Response_builder(builder, obj)
  File "/glob/development-tools/versions/oneapi/2023.0.1/oneapi/intelpython/latest/lib/python3.9/site-packages/conda/auxlib/logz.py", line 135, in requests_models_Response_builder
    content = dumps(resp, indent=2)
  File "/glob/development-tools/versions/oneapi/2023.0.1/oneapi/intelpython/latest/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/glob/development-tools/versions/oneapi/2023.0.1/oneapi/intelpython/latest/lib/python3.9/json/encoder.py", line 201, in encode
    chunks = list(chunks)
MemoryError
failed

CondaMemoryError: The conda process ran out of memory. Increase system memory and/or try again.


Process exceeded login node resource limit.
Please run jobs on compute nodes using qsub or start an interactive shell using qsub -I.
Please visit the forums at: https://software.intel.com/en-us/forums/intel-devcloud```